app/controllers/users_controller.rb

```diff
@@ -65,6 +65,8 @@ def create
   def create_external
     @user = User.new(params[:user])
     @user.password = generate_new_password
+    @user.created_by = current_user.try(:id)
```

Comment:
@noelrappin looks like you dropped the DB column but missed this reference, making your tests sad.